### PR TITLE
Fix circular saw spam

### LINKED ISSUE #18224: Circular Saw spamming "You don't have guns with long barrels" when using it to remove things from vehicles
After turning the circular saw on, using it to 'remove' in the vehicle build menu first says "Cut up metal where? (Direction Button)" then spams "You dont have guns with long barrels" til the batteries run out.

### PR DESCRIPTION
**Pull request by sethsimon**
Fixes #18224.

### CODE REVIEW

File: src/iuse.cpp [hunk 1]
```diff
@@ -7874,9 +7878,9 @@ int iuse::ladder( player *p, item *, bool, const tripoint& )
     return 1;
 }
 
-int iuse::saw_barrel( player *p, item *, bool, const tripoint& )
+int iuse::saw_barrel( player *p, item *, bool ticking, const tripoint& )
```
Comment (BevapDin) [inline src/iuse.cpp]: Why is the parameter here named differently? It's named `t` in the other `iuse` functions. It should be named the same in all of them, for consistency, as it means the same in all of them.

**Comment by Coolthulhu** [comment]
Still, `ticking` should be renamed for consistency.

**Comment by sethsimon** [comment]
It seemed like it would be a bad usage of a one-letter variable name. Since it's just a `bool`, it isn't clear what it's supposed to mean, and the context (body of the function) usually doesn't help much. I agree with consistency, but maybe all the others should be renamed instead.

**Comment by Coolthulhu** [comment]
Without a comment, `ticking` isn't descriptive enough. In such a case, consistency is preferred.

**Comment by sethsimon** [comment]
It isn't? It seems pretty descriptive to me. But on the other hand, so does anything once you understand it. Since that's the best name I can come up with, I'll make a PR to rename it (and the other instance, where it's called `active`).

**Pull request merged**